test-cases/gemini/gemini-3h-with-nemesis.yaml

```diff
@@ -21,7 +21,7 @@ gemini_cmd: "gemini -d --duration 3h --warmup 30m \
 --replication-strategy \"{'class': 'SimpleStrategy', 'replication_factor': '3'}\" \
 --oracle-replication-strategy \"{'class': 'SimpleStrategy', 'replication_factor': '1'}\""
 
-gemini_version: 'latest'
+gemini_version: '1.7.5'
```

Comment:
we care only about this test cases ? i.e. all other gemini test are configured with `latest`

Reply:
That was the Gemini content of 2021.1.13 (https://docs.google.com/document/d/1OOILCRfdQ6fg9LWpAtjVuw2fU1Bomtge7CssTWdgZKU/edit )
Perhaps @roydahan can confirm that.

Reply:
For 2021.1 we can pin it, it's not a solution but will reduce some noise for now.